### PR TITLE
created card component, landing page, navbar and sidebar

### PR DESCRIPTION
**Pull request by Sharmaa-Ansh**

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
A couple of pointers
- Use rem for size values and keep the values consistent
- Store the component css files in a common file and import it everywhere

File: Components/Card/Card.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    margin-left: 1rem;
+    margin-top: 1.1rem;
+    margin-right: 0.5rem;
+}
+
+.CardWithImage{
+    width: 16rem;
+    height: 25rem;
+    box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%);
+
```
Comment (anuj-kumary) [inline Components/Card/Card.css:30]: Remove this extra spaces

File: Index.html [hunk 2]
```diff
@@ -4,9 +4,54 @@
...
+            <div class="Page__Infobox">
+                <h1>Build Your own U.I. </h1>
+                <p>Copy and paste the components at one click. It is user-friendly and customizable. Spend less time in designing and concentrate more on logic without hustling on UI.</p>
+                <div class="CTA-container">
+                    <a href="" class="btn btn--Primary">Get Started </a>
+
+                </div>
+            </div>
+            <div class="Page__Image">
+                <img class="Img-Full" src="/Assets/undraw_design_components_9vy6.svg" alt="">
```
Comment (anuj-kumary) [inline Index.html:43]: give some value to alt

File: css/Index.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+.Page__Infobox{
+    border: solid 1px;
+    min-width: 50%;
+    font-size: 1.5rem;
+    padding-right: 1rem;
+    /* display: flex;
+    flex-direction: column;
+    align-items: center; */
```
Comment (anuj-kumary) [inline css/Index.css:22]: Remove this unwanted code

**Pull request merged**